topics:
  outgoing:
    laserscan: laser_scan

comm_settings:
  networking:
    sensor:
      ip: 192.168.1.10
      tcp_port: 2122
    host:
      ip: 192.168.1.9
      udp_port: 6060
  channel: 0
  enabled: true
  e_interface_type: 0
  publish_frequency: 1
  angle_start: 0
  angle_end: 0
  features:
    general_system_state: true 
    derived_settings: true
    measurement_data: true
    intrusion_data: true
    application_io_data: true



